dev-requirements.txt

```diff
@@ -1,13 +0,0 @@
-# Dev/Deployment
-sphinx==2.2.1
-sphinx_rtd_theme==0.4.3
-sphinx-autobuild==0.7.1
-sphinx-click==2.3.1
-gitpython>=2.1.0
-nose
-codecov
-coverage
-pypi-publisher
-scikit-learn
-scipy
-kubernetes
```

Comment:
These packages are included in either `install_requires` or `small-requirements.txt`.